examples/python/human_pose_tracking/main.py

```diff
@@ -60,7 +60,7 @@
 """.strip()
 
 
-def track_pose(video_path: str, segment: bool) -> None:
+def track_pose(video_path: str, segment: bool, max_frame_count: int | None) -> None:
```

Comment:
Good habit to get into:

```suggestion
def track_pose(video_path: str, *, segment: bool, max_frame_count: int | None) -> None:
```